### PR TITLE
fix: Update canonical reference link on _app.tsx

### PR DESCRIPTION
**Pull request by s-n-park**
We want to fix the canonical link reference for SEO

See this issue for more details:
https://github.com/near/near-discovery/issues/1029

### CODE REVIEW

**Comment by shelegdmitriy** [comment]
Hey @s-n-park. Thanks for your contribution! May I ask you to run 

```
pnpm prettier:write
```
and commit changes to fix CI / Prettier workflow please?

**Comment by s-n-park** [comment]
Thanks for the feedback @shelegdmitriy! Just ran that, can you let me know if there's anything else here you need from me?

**Comment by shelegdmitriy** [comment]
Are we both good with this PR @charleslavon ?

File: src/pages/_app.tsx [hunk 1]
```diff
@@ -100,11 +100,7 @@ export default function App({ Component, pageProps }: AppPropsWithLayout) {
       <Head>
         <meta name="google-site-verification" content="CDEVFlJTyVZ2vM7ePugKgWsl_7Rd-MrfDv42u0vZ0B0" />
         <link rel="icon" href="favicon.ico" />
-        <link
-          rel="canonical"
-          href={`${process.env.NEXT_PUBLIC_HOSTNAME}/near/widget/NearOrg.HomePage`}
-          key="canonical"
-        />
+        <link rel="canonical" href={`${process.env.NEXT_PUBLIC_HOSTNAME}${router.asPath}`} key="canonical" />
```
Comment (charleslavon) [inline src/pages/_app.tsx:103]: @s-n-park do we need a `/` in between these two variables? That pattern is used in the diff above

Comment (charleslavon) [reply src/pages/_app.tsx:103]: actually, the preview looks ok.

**Review by charleslavon — APPROVED** [review]

**Pull request merged**